docs/DafnyRef/Statements.md

```diff
@@ -1828,10 +1895,10 @@ possible where there might be information overload otherwise.
 
 But then there may be specific instances where the definition of that opaque function is needed. In that situation, the
 body of the function can be _revealed_ using the reveal statement. Here is an example:
-```dafny
+```dafny <!-- %check-verify Statements.9.expect -->
 function {:opaque} f(i: int): int { i + 1 }
 
-method m(int i) {
```

Comment:
Someone was still writing C#. :)

Reply:
Java actually - 20 years of habit die hard